index.js

```diff
@@ -57,10 +60,14 @@ module.exports = (api, { pluginOptions = {} }) => {
     server.buildStatic({
       packageJson: {
         name: '@storybook/vue',
-        version: '4.0.0-alpha.20',
+        version: '^4.0.0',
       },
-      wrapInitialConfig: wrapInitialConfig(api, options),
-      wrapDefaultConfig,
+      frameworkPresets: [
+        {
+          preset: require.resolve('./preset'),
```

Comment:
The key for the preset path should be `name` not `preset`
```suggestion
          name: require.resolve('./preset'),
```

See https://github.com/storybooks/storybook/blob/master/lib/core/src/server/presets.js#L22